test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -2155,8 +2152,8 @@ Child manifest is named entry:
 `;
 
 exports[`StatsTestCases should print correct stats for runtime-chunk-issue-7382 1`] = `
-"Entrypoint e1 = runtime.js all.js e1.js
```

Comment:
Same here. e1 should be kept. It's the only difference between both entrypoints.